src/modules/previewpane/SvgPreviewHandler/SvgPreviewControl.cs

```diff
@@ -162,6 +167,16 @@ private void FormResized(object sender, EventArgs e)
             }
         }
 
+        // Disable loading resources.
+        private void CoreWebView2_BlockExternalResources(object sender, CoreWebView2WebResourceRequestedEventArgs e)
+        {
+            // Show local file we've saved with the svg contents. Block all else.
+            if (new Uri(e.Request.Uri) != _localFileURI)
+            {
+                e.Response = _browser.CoreWebView2.Environment.CreateWebResourceResponse(null, 404, "Not found", null);
```

Comment:
what about 403 (forbidden). makes more sense for me.

Reply:
I think you're right. 403 makes more sense here and I think it's more up to spec.